maskhit/train.py

```diff
@@ -1,17 +1,29 @@
+"""
+Usage: python train.py --study=ibd --mil1=vit_h8l12 --mil2=ap --num-patches=0 

```

Comment:
Please try generalizing this. Then you can add your code as an example, or make a bash script and paste this command instead, so other people won't encounter doc specific to your dataset.